docs/find_and_replace_action_json_patch.md

```diff
@@ -0,0 +1,597 @@
+
+JSON Patch uses `JSON Pointers` as arguments in all of these operations to target a specific key/value of your JSON.
+
+Using the following JSON Snippet as example:
+
+```JSON
+"subject_lcgft_terms": [
+        {
+            "uri": "http:\/\/id.loc.gov\/authorities\/genreForms\/gf2017027249",
+            "label": "Photographs"
+        }
+]
+```
+
+These JSON Pointers will resolve in the following manner:
+
+- `/subject_lcgft_terms` 
+
+```JSON
+[
+    {
+            "uri": "http:\/\/id.loc.gov\/authorities\/genreForms\/gf2017027249",
+            "label": "Photographs"
+    } 
+]
+```
+
+ - `/subject_lcgft_terms/0` 
+ 
+```JSON
+{
+    "uri": "http:\/\/id.loc.gov\/authorities\/genreForms\/gf2017027249",
+    "label": "Photographs"
+}
+```
+
+-  `/subject_lcgft_terms/0/label`
+
+```
+Photographs
+```
+
+AS you can see a JSON Pointer is very precise , allowing you to target complete structures and values but it *does not allow* Wildcard Operations. Means you can not "search" or do loosy comparissons. This very fact limits many times the use case. E.g. if you have a list of terms like this:
+
+```JSON
+"terms": [
+	"term1",
+	"term2",
+	"term3"
+]
+```
+
+and you want to "test" for the existence of `"term3"` before applying a change, you would need to know exactly at what position inside the `terms Array` (Starting from 0) it will/should be found.  And that might not be consistent across every ADO. 
+
+So how do we use these pointers in an operation inside a JSON Patch document? Using the same fake "terms" JSON snippet the previously listed, operations are written like this:
+
+## add
+
+```JSON
+{ "op": "add", "path": "/terms/0", "value": "term_another" }
+```
+
+This will add before the first term (in this case "term1" ) "term_another" as a value.
+
+??? info "At the end"
+
+    you can use a dash (`-`) (e.g. "/terms/-") instead of the numeric position of an array entry to denote that the "value" needs to be added at the end. This is needed specially for empty lists. You can not target `0` position on an empty array.
+
+## remove
+
+```JSON
+{ "op": "remove", "path": "/terms/1"}
+```
+
+This will remove the second term  (in this case "term2" )
+
+## replace
+
+```JSON
+{ "op": "replace", "path": "/terms/1", "value": "term_again" }
+```
+This will remove the second term  (in this case "term2" ) and put in its place "term_again" as a value. Basically two operationes, “remove“ and “add “ in a single step
+
+## move
+
+```JSON
+{ "op": "move", "from": "/terms", "path": "/terms_somewhere_else"}
+```
+
+This will copy  all values inside top JSON key `terms` into a new top JSON key named `terms_somewhere_else` and remove then the old `terms` key.
+
+## copy
+
+```JSON
+{ "op": "copy", "from": "/terms", "path": "/terms_somewhere_else"}
+```
+
+Similar to “move“, it will copylvalues inside top JSON key `terms` into a new top JSON key named `terms_somewhere_else` without removing then the old `terms` key or its content!
```

Comment:
```suggestion
Similar to “move“, it will copy values inside top JSON key `terms` into a new top JSON key named `terms_somewhere_else` without removing then the old `terms` key or its content!
```
Typo.